gui.lua

```diff
@@ -596,6 +596,9 @@ local function addCityOverview(city, anchor)
     -- citizen count
     tbl.add{type = "label", caption = {"", {"tycoon-gui-citizens"}, ": "}}
     tbl.add{type = "label", caption = {"", countCitizens(city)}}
+    -- debt
+    tbl.add{type = "label", caption = {"", {"tycoon-gui-debt"}, ": "}}
```

Comment:
I'm wondering if this will be easy enough to understand for players. Shall we include a description sentence along the lines of "this is how much the city owes you, make space in treasuries"?

Reply:
additional description would be really helpful, any example of how it's done? i'm eager for settings' `?` round hint

i just can't get proper name for this: "pending payments" or just "unpaid"
first naming was even simpler `Debt:`, but it doesn't mean more for players - "debt of what and to where?" questions will raise